targets/f7/api_symbols.csv

```diff
@@ -20,6 +20,7 @@ Header,+,applications/services/gui/modules/file_browser.h,,
 Header,+,applications/services/gui/modules/file_browser_worker.h,,
 Header,+,applications/services/gui/modules/loading.h,,
```

Comment:
Bump minor api version

Reply:
Not sure if I'm doing this correctly... I assume I need to run `.\fbt` , fix entries with ? and then commit?

Reply:
Change `Version,+,68.1,,` to `Version,+,68.2,,`